docs/integrations/google_colab.rst

```diff
@@ -3,38 +3,129 @@
+to Neptune and refer to them even after the Colab kernel has died.
 
-    .. code-block:: Bash
+Introduction
+------------
 
-        pip install neptune-client
+This guide will show you how to:
 
-2. Set Neptune API token:
+#. Install ``neptune-client``
+#. Connect Neptune to your notebook and create the first experiment
+#. Log simple metrics to Neptune and explore them in the Neptune UI
 
-    Go to the Neptune web app and `get your API token <../python-api/how-to/organize.html#find-my-neptune-api-token>`_. Set it to the environment variable `NEPTUNE_API_TOKEN`:
+Before you start
+----------------
 
-    .. code-block:: Bash
+Make sure that you have an account with both |Google| and |Neptune|.
 
-        env NEPTUNE_API_TOKEN='your_private_neptune_api_token=='
+Quickstart
+----------
 
-3. Delete this cell.
+|Run on Colab|
 
-    .. warning::
+Step 1: Install Neptune client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Go to your first cell in Google Colab and install ``neptune-client``:
 
-        It is very important that you delete this cell, so as not to share your private token with anyone.
+.. code-block:: Bash
 
-4. Now, run your training script with Neptune:
+   pip install neptune-client
 
-    .. code-block:: Python
+Step 2: Set Neptune API token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-        import neptune
-        neptune.init('USER_NAME/PROJECT_NAME')
+.. image:: ../../_static/images/integrations/getpass.gif
+  :target: ../../_static/images/integrations/getpass.gif
+  :alt: Set API token using getpass()
+  
+#. Go to the Neptune web app and get your API token
+   
+#. Run the code below:
+    
+   .. code-block:: Python
 
-        with neptune.create_experiment():
-            neptune.send_metric('auc', 0.92)
\ No newline at end of file
+      from getpass import getpass
+      api_token = getpass('Enter your private Neptune API token: ')
+
+#. Enter the token in the input box. This will save your token to ``api_token``.
+
+
+Step 3: Initialize your Neptune Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
+
+.. code-block:: Python
+
+    import neptune
+
+    neptune.init(project_qualified_name = 'your_user_name/your_project_name'
+                 , api_token = api_token)
+
+Step 4: Run your training script with Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
+
+.. code-block:: Python
+
+    from numpy import random
+    from time import sleep
+
+    neptune.create_experiment()
+        
+    neptune.log_metric('single_metric', 0.62)
+        
+    for i in range(100):
+        sleep(0.2) # to see logging live
+        neptune.log_metric('random_training_metric', i*random.random())
+        neptune.log_metric('other_random_training_metric', 0.5*i*random.random())
+	
+    neptune.stop()
+
+Step 5: Check metrics on the Neptune UI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Follow the link shown to view your experiment progress and metrics in the Neptune UI.
+
+|Explore experiment|
+
```

Comment:
I'd add a section with advanced options showing:

- how to download model checkpoint (or other file) from Neptune directly into Colab
- how to analyze your Neptune experiments data by downloading the dashboard 

both of those sections are explained here https://docs.neptune.ai/logging-and-managing-experiment-results/downloading-experiment-data-programmatically/index.html

Reply:
I've a link to this doc under the "What's next" section.

Do you want me to elaborate these under a new "Advanced topics" section? I feel that would be duplication of work and will need more documentation changes if the process/UI is changed in the future.